x-pack/plugins/security_solution/public/management/links.ts

```diff
@@ -215,11 +215,11 @@ export const getManagementFilteredLinks = async (
   plugins: StartPlugins
 ): Promise<LinkItem> => {
   try {
-    const currentUserResponse = await plugins.security.authc.getCurrentUser();
+    const currentUserResponse = await plugins.security?.authc.getCurrentUser();
```

Comment:
nit: It turns out that the security plugin will always be defined even if security is disabled within Elasticsearch. I think we could probably leave this the way it was.

Reply:
How is that possible? I thought that if you put it in the `kibana.json` as optional then it maybe not be there.

Reply:
Yeah that's what I thought too, but it's no longer the case. I think it's because you can't actually disable the security plugin itself. It is always enabled. You can disable security though, but it is now done within Elasticsearch directly. So for example if you attempt to call the suggestions API, Elasticsearch will throw a 500 I think saying that the API does not exist.

That's why we now have to check for the license to determine if security is disabled: https://github.com/elastic/kibana/blob/main/x-pack/plugins/cases/server/services/user_profiles/index.ts#L109
https://github.com/elastic/kibana/issues/137861 -> I still need to clean up the `undefined` checks we have in cases.